Cargo.toml

```diff
@@ -1,36 +1,21 @@
-[package]
-name = "rngs"
-version = "0.0.0"
-authors = ["The Rand Project Developers", "The Rust Project Developers"]
-license = "MIT OR Apache-2.0"
-readme = "README.md"
-repository = "https://github.com/rust-random/rngs"
-description = """
-Extra random number generators
-"""
-autobenches = true
-edition = "2018"
-publish = false
-
 [workspace]
 members = [
+    "benches",
     "rand_jitter",
     "rand_isaac",
     "rand_xorshift",
     "rand_xoshiro",
     "rand_hc",
 ]
+resolver = "2"
 
-[dev-dependencies]
-# Only for benches:
-criterion = "0.3.5"
-criterion-cycles-per-byte = "0.1.2"
-rand_core = { version = "0.6", features = ["getrandom"] }
-rand_xoshiro = { path = "rand_xoshiro", version = "0.6" }
-rand_isaac = { path = "rand_isaac", version = "0.3" }
-rand_xorshift = { path = "rand_xorshift", version = "0.3" }
-rand_hc = { path = "rand_hc", version = "0.3" }
-
-[[bench]]
-name = "mod"
-harness = false
+[workspace.package]
+authors = ["The Rand Project Developers", "The Rust Project Developers"]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/rust-random/rngs"
```

Comment:
I don't think we need this section in the virtual manifest file. We don't (and probably should not) use inherited fields in the crates.